.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,86 @@
+name: CI Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # run every Monday at 5am UTC
+    - cron: '0 5 * * 1'
+
+env:
+  TOXARGS: '-v'
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    if: "!(contains(github.event.head_commit.message, '[skip ci]') || contains(github.event.head_commit.message, '[ci skip]'))"
```

Comment:
This incantation only works for push event, not pull request event.

Reply:
thanks, I suppose we can just use your action instead of this hackery.